variables.tf

```diff
@@ -59,3 +59,33 @@ variable "install_security_agent" {
   default = false
   description = "Optional flag for installation of security agent (https://docs.cast.ai/product-overview/console/security-insights/)"
 }
+
+variable "agent_values" {
+  description = "List of YAML formatted string with agent values"
```

Comment:
The description does not sound nice in english. Should it be something like "List of YAML formatted string values for xyz" ?